includes/abilities-api/class-wp-ability.php

```diff
@@ -188,7 +189,6 @@ public function __construct( string $name, array $args ) {
 	 *   output_schema?: array<string,mixed>,
 	 *   annotations?: array<string,mixed>,
```

Comment:
This should probably also be changed inside WP_Abilities_Registry

```suggestion
	 *   annotations?: array<string,(bool|string)>,
```